wordpress-mobile/WordPress-iOS
# Don't show non-`.blog` domain suggestions for Blogger plans

Fixes #12063, an issue where we'd suggest a non-`.blog` domain suggestions for Blogger plan users, who only get `.blog` for free. This fixes it!

To test:

1. Make sure you have a site with Blogger plan and a domain credit
1. Open the app 
1. Go to the Blog Detail view
1. Tap on the Domain redemption banner
1. Verify that there's only .blog domains visible

This PR also has a WPKit counterpart: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/168

Here are some screenshots showing a blogger and non-blogger plans:

| non-Blogger plan | Blogger plan |
| ----- | ----- |
| ![Simulator Screen Shot - iPhone Xs Max - 2019-07-16 at 11 20 26](https://user-images.githubusercontent.com/1594081/61283720-d7a3bb00-a7bd-11e9-8f83-b442d5ecab1e.png) | ![Simulator Screen Shot - iPhone Xs Max - 2019-07-16 at 10 50 07](https://user-images.githubusercontent.com/1594081/61283722-d83c5180-a7bd-11e9-8027-c6a7ea2c310b.png) |
